### PR TITLE
 Azure SDK for PowerBI Dedicated

### PR DESCRIPTION
**Pull request by selasagi**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [V] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [V] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [V] Title of the pull request is clear and informative.
- [V] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [V] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [V] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [V] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [V] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Review by shahabhijeet — CHANGES_REQUESTED** [review]
Missing description. This PR cannot be reviewed if there is no information that describes what this PR is about.
Add detail description about the change
Link to spec PR
make sure CI is passing.

**Comment by selasagi** [comment]
**Description:**
First generation of SDK for Power BI Dedicated resource. New Azure resource which simplify the way ISVs and developers use Power BI capabilities.
**Link to spec PR:** 
https://github.com/Azure/azure-rest-api-specs/tree/current/specification/powerbidedicated/resource-manager/Microsoft.PowerBIdedicated/2017-10-01/powerbidedicated.json

Thanks!

File: src/SDKs/PowerBIDedicated/Management.PowerBIDedicated/Microsoft.Azure.Management.PowerBIDedicated.csproj [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  <PropertyGroup>
+    <TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
+  </PropertyGroup>
+
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net452' ">
+    <Reference Include="System" />
+    <Reference Include="Microsoft.CSharp" />
+  </ItemGroup>
+
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard1.4' ">
```
Comment (shahabhijeet) [inline src/SDKs/PowerBIDedicated/Management.PowerBIDedicated/Microsoft.Azure.Management.PowerBIDedicated.csproj]: @selasagi you don't need any of these references. Please model your sdk project file by looking at other project files in the repo, e.g. https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/BatchAI/Management.BatchAI/Microsoft.Azure.Management.BatchAI.csproj

File: src/SDKs/PowerBIDedicated/Management.PowerBIDedicated/Microsoft.Azure.Management.PowerBIDedicated.csproj [hunk 2]
```diff
@@ -0,0 +1,44 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <PropertyGroup>
+    <Description>Provides management capabilities for Azure PowerBI Dedicated accounts.</Description>
+    <AssemblyTitle>Microsoft Azure PowerBI Dedicated Management Library</AssemblyTitle>
+    <VersionPrefix>2.0.1</VersionPrefix>
+    <AssemblyName>Microsoft.Azure.Management.PowerBIDedicated</AssemblyName>
+    <PackageId>Microsoft.Azure.Management.PowerBIDedicated</PackageId>
+    <PackageTags>Microsoft Azure resource management;asazure;REST HTTP client;azureofficial;windowsazureofficial;netcore451511</PackageTags>
```
Comment (shahabhijeet) [inline src/SDKs/PowerBIDedicated/Management.PowerBIDedicated/Microsoft.Azure.Management.PowerBIDedicated.csproj]: @selasagi also add PackageReleaseNotes for your nuget package.
As this is your first version for this brand new service, start with 0.9.0-preview
Any questions reach out to me.

**Review by shahabhijeet — CHANGES_REQUESTED** [review]

**Pull request closed without merging**